WooFoundation/WooFoundation/Colors/UIColor+SemanticColors.swift

```diff
@@ -427,6 +427,12 @@ public extension UIColor {
             return .secondarySystemGroupedBackground
         }
     }
+
+    /// Products tab list cell selected background color
+    ///
+    static var productsCellSelectedBackgroundColor: UIColor {
+        return UIColor.wooCommercePurple(.shade0)
```

Comment:
Just a note that when you use `UIColor.wooCommercePurple(.shade0)` it returns shade0 in light mode and shade100 in dark mode. Is that the expected behavior?

I didn't see any discussion about dark mode so I just wanted to double check. Here's what I'm seeing:


https://github.com/woocommerce/woocommerce-ios/assets/8658164/405b6641-46d2-4091-93c3-e827dec63b3c

Reply:
@joshheald what do you think, I see that shade0 is mentioned in https://github.com/woocommerce/woocommerce-ios/issues/11659 so we wanted to double check regarding light/dark mode

Reply:
`shade100` in dark mode is fine for now, we can merge and ship with that. 

@joe-keenan – when you're back, could you let us know whether you've got a better option for the dark mode row selection colour please?